crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -109,6 +112,9 @@ def end_step(success = true, errors = {})
         }
         next_step
         save
+        if finished? && success
+          FileUtils.touch("/var/lib/crowbar/upgrade/6-to-7-done")
```

Comment:
the installer files are all in the schema `crowbar-installed-ok` or `crowbar-install-failed`, shall we stick to this schema? that would be `6-to-7-upgraded-ok`